### PR TITLE
Rename tsa bundle to RFC3161timestamp

### PR DESCRIPTION
**Pull request by hectorj2f**
Signed-off-by: Hector Fernandez <hector@chainguard.dev>

#### Summary

As discussed, we decided to rename the TSABundle to RFC3161Timestamp as it represents better its data used for signing and verification.

### CODE REVIEW

File: pkg/oci/internal/signature/layer.go [hunk 1]
```diff
@@ -29,11 +29,11 @@ import (
...
-	sigkey       = "dev.cosignproject.cosign/signature"
-	certkey      = "dev.sigstore.cosign/certificate"
-	chainkey     = "dev.sigstore.cosign/chain"
-	BundleKey    = "dev.sigstore.cosign/bundle"
-	TSABundleKey = "dev.sigstore.cosign/3161timestamp"
+	sigkey              = "dev.cosignproject.cosign/signature"
+	certkey             = "dev.sigstore.cosign/certificate"
+	chainkey            = "dev.sigstore.cosign/chain"
+	BundleKey           = "dev.sigstore.cosign/bundle"
+	RFC3161TimestampKey = "dev.sigstore.cosign/3161timestamp"
```
Comment (asraa) [inline pkg/oci/internal/signature/layer.go:36]: should we make this rfc3161?

**Review by asraa — COMMENTED** [review]
small comment about the annotation name, otherwise lgtm

File: pkg/oci/internal/signature/layer.go (hunk 1 above)
Comment (hectorj2f) [reply pkg/oci/internal/signature/layer.go:36]: yes, I was thinking the same. 👍🏻

Comment (hectorj2f) [reply pkg/oci/internal/signature/layer.go:36]: Fixed!

**Review by priyawadhwa — APPROVED** [review]

**Pull request merged**